src/components/ConfirmationModal.tsx

```diff
@@ -54,35 +50,30 @@ const Label = styled.Text<{ light: boolean }>`
 
 export interface ConfirmationModalPropsType {
   visible: boolean
-  setIsModalVisible: Function
-  navigation: StackNavigationProp<RoutesParamsType, 'WordChoiceExercise' | 'ArticleChoiceExercise' | 'WriteExercise'>
-  route: RouteProp<RoutesParamsType, 'WordChoiceExercise' | 'ArticleChoiceExercise' | 'WriteExercise'>
+  setVisible: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
Why do we need this type here? Just `(visible: boolean) => void` should be enough.

Reply:
Cause @f1sh1918 requested this change, will revert.